src/features/contracts/contracts-directory.ts

```diff
@@ -81,7 +81,9 @@ const normalizeAssets = async (
 };
 
 const getArchiveLinkForRepo = (repo: RepositoryInfo) => {
-	return `https://api.github.com/repos/${repo.owner}/${repo.name}/tarball`;
+	return `https://api.github.com/repos/${repo.owner}/${repo.name}/tarball/${
+		repo.branch || ''
```

Comment:
Most (if not all) `|| ''` could probably be `?? ''` but not a blocker.

Reply:
Ah nice catch. Fixed this in the latest push